addons/crm/views/crm_forecast_views.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="crm_lead_view_kanban_forecast" model="ir.ui.view">
+        <field name="name">crm.lead.view.kanban.forecast</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_case_kanban_view_leads"/>
+        <field name="priority">2</field>
```

Comment:
Why? The view is requested explicitely by its action, and this is higher than the view it extends.